src/udisksdaemonutil.c

```diff
@@ -56,6 +56,22 @@
  * Various utility routines.
  */
 
+
+/**
+ * udisks_string_wipe_and_free:
+ * @string: A string with potentially unsafe content or %NULL.
+ *
+ * Wipes the buffer and frees the string.
+ */
+void udisks_string_wipe_and_free (GString *string)
+{
+  if (string != NULL)
+    {
+      memset (string->str, '\0', string->len);
+      g_string_free (string, TRUE);
+    }
+}
+
```

Comment:
Please make this new function part of the commit that uses it. I think it might be best to just squash this commit together with the one that requires these changes. That would also help if we ever want to run ``git bisect``.